src/AutoMapper.Collection.Tests/MapCollectionWithEqualityTests.cs

```diff
@@ -118,14 +118,14 @@ public void Should_Be_Fast_With_Large_Lists_Cant_Extract_Negative()
             {
                 x.AddCollectionMappers();
                 // ReSharper disable once NegativeEqualityExpression
-                x.CreateMap<ThingDto, Thing>().EqualityComparison((ThingDto dto, Thing entity) => !(dto.ID != entity.ID));
+                x.CreateMap<ThingDto, Thing>().EqualityComparison((ThingDto dto, Thing entity) => dto.ID == entity.ID);
```

Comment:
Regarding to the comments and the name of the test this expression should be double negative.